src/components/index.js

```diff
@@ -2,3 +2,5 @@ export { Navbar } from "./Navbar/Navbar";
 export { CategoriesCard } from "./CategoriesCard/CategoriesCard";
 export { UpcomingsCard } from "./UpcomingsCard/UpcomingsCard";
 export { Footer } from "./Footer/Footer";
+export { Filter } from "./Filter/Filter";
```

Comment:
nice that you have created index.js for all imports.

Reply:
Thank you